Общие подкаталоги: openssh-portable-V_9_9_P1/contrib и openssh-portable-V_9_9_P1.mod/contrib
Общие подкаталоги: openssh-portable-V_9_9_P1/.github и openssh-portable-V_9_9_P1.mod/.github
Общие подкаталоги: openssh-portable-V_9_9_P1/m4 и openssh-portable-V_9_9_P1.mod/m4
Общие подкаталоги: openssh-portable-V_9_9_P1/openbsd-compat и openssh-portable-V_9_9_P1.mod/openbsd-compat
Общие подкаталоги: openssh-portable-V_9_9_P1/regress и openssh-portable-V_9_9_P1.mod/regress
diff -uN openssh-portable-V_9_9_P1/ssh_config openssh-portable-V_9_9_P1.mod/ssh_config
--- openssh-portable-V_9_9_P1/ssh_config	2024-09-20 01:20:48.000000000 +0300
+++ openssh-portable-V_9_9_P1.mod/ssh_config	2024-09-20 12:38:30.117550815 +0300
@@ -44,3 +44,5 @@
 #   ProxyCommand ssh -q -W %h:%p gateway.example.com
 #   RekeyLimit 1G 1h
 #   UserKnownHostsFile ~/.ssh/known_hosts.d/%k
+
+SendEnv LANG
